examples/eks-cluster-with-new-vpc-and-efs/README.md

```diff
@@ -0,0 +1,164 @@
+# EKS Cluster Deployment with new VPC and EFS
+This example deploys the following Basic EKS Cluster with VPC
+ - Creates a new sample VPC, 3 Private Subnets and 3 Public Subnets
+ - Creates Internet gateway for Public Subnets and NAT Gateway for Private Subnets
+ - Creates EKS Cluster Control plane with one managed node group and fargate profile
+ - Creates EFS file system for backing the dynamic provisioning of persistent volumes
+
+## How to Deploy
+### Prerequisites:
+Ensure that you have installed the following tools in your Mac or Windows Laptop before start working with this module and run Terraform Plan and Apply
+1. [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html)
+3. [Kubectl](https://Kubernetes.io/docs/tasks/tools/)
+4. [Terraform](https://learn.hashicorp.com/tutorials/terraform/install-cli)
+
+### Deployment Steps
+#### Step1: Clone the repo using the command below
+
+```shell script
+git clone https://github.com/aws-samples/aws-eks-accelerator-for-terraform.git
+```
+
+#### Step2: Run Terraform INIT
+Initialize a working directory with configuration files
+
+```shell script
+cd examples/eks-cluster-with-new-vpc-and-efs/
```

Comment:
Could you rename this example folder from `eks-cluster-with-new-vpc-and-efs` to `aws-efs-csi-driver`. Update the README once the renaming is done.

We are planning to rename other example folders as well to easily recognise examples.